### DISCUSSION TITLE
Who do I contact for personal data removal in commit?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#87016] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I wrote an email to privacy@github.com with a request to remove my personal data from a commit inside a public repository one week ago. I haven't received any response since then. Is it the right email? Who should I contact in such a case?

**Comment by W4LD3V** [comment]
To remove your personal data from a commit in a public GitHub repository:

1. **Contact GitHub Support**: Submit your request via GitHub's official support page at [https://support.github.com/contact](https://support.github.com/contact). Be specific about the issue, providing details like the repository name and commit identifier.

2. **Follow-Up**: If you don't receive a response within a few business days, send a follow-up inquiry through the same channel.

3. **Keep Records**: Document your communications with GitHub for future reference. 

Remember, the process may take some time due to GitHub's policies on modifying public repository histories.
